### PR TITLE
Prepare 1.2.0

### PR DESCRIPTION
**Pull request by Natim**
* [x] Update changelog
* [x] Bump version in `setup.py` and `docs/conf.py`
* [ ] Tag
* [ ] Publish on Pypi

r? @almet

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+1.2.0 (2016-01-18)
 ==================
 
-- Nothing changed yet.
+- Adding the ability to define services imperatively. (#335)
+- Clean cornice/statics/ files. (#345)
+
+Bug fixes
+
+- Convert None to null before calling deserialize. (#342)
```
Comment (almet) [inline CHANGES.txt]: This is unclear: What is this for?

Comment (Natim) [inline CHANGES.txt]: I don't remember but you merged it some time ago, see #342

Comment (almet) [inline CHANGES.txt]: Consider replacing the entry in the `CHANGES.txt` by "Convert `None` to `colander.null` before calling colander's `deserialize` function."

Comment (Natim) [inline CHANGES.txt]: Ok.

**Comment by almet** [comment]
It's all right! You can release :)

**Pull request merged**